src/NET/MvvmDialogs/DialogTypeLocators/NamingConventionDialogTypeLocator.cs

```diff
@@ -32,14 +33,20 @@ public Type Locate(INotifyPropertyChanged viewModel)
             Type viewModelType = viewModel.GetType();
 
             Type dialogType = Cache.Get(viewModelType);
+
             if (dialogType != null)
             {
                 return dialogType;
             }
 
             string dialogName = GetDialogName(viewModelType);
-            
+
             dialogType = GetAssemblyFromType(viewModelType).GetType(dialogName);
+
+            // if view name ends with "View"
+            if(dialogType == null)
```

Comment:
I don't think this adheres to the common MVVM naming convention. If your views are suffixed with `View`, then probably you should write your own `IDialogTypeLocator`? I could possibly rewrite this class in a way that would make it easy for you inherit this class and override some of the methods, what do you think about that?